wazuh/wazuh-dashboard-plugins
# Hide vulnerabilities global filter editor button

### Description
This pull request hides the global filter editor button in the Vulnerabilities dashboard and inventory tab. This prevents the user from disabling implicit filters.

![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/814a6865-812d-4eaa-af51-fd64aebf5ab0)

 
### Issues Resolved
Closes #6654

### Evidence

![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/e7ebed36-31dd-4614-bc1b-7e0a96e1e6e6)

![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/535e1a7f-1e13-44ba-b9e4-9a6c8bf80eb1)


### Test
- Go to _Threat Intelligence / Vulnerability detector / Dashboard_
- Verify the global filter editor is hidden
- Go to _Threat Intelligence / Vulnerability detector / Inventory_
- Verify the global filter editor is hidden

### Check List
